src/bundle/ui-dev/src/modules/universal-discovery/components/content-type-selector/content.type.selector.js

```diff
@@ -2,11 +2,9 @@ import React, { useState, useContext } from 'react';
 
 import { createCssClassNames } from '../../../common/helpers/css.class.names';
 import { SelectedContentTypesContext } from '../search/search';
-import { AllowedContentTypesContext } from '../../universal.discovery.module';
```

Comment:
I'm not convinced of this approach.
`AllowedContentTypesContext` stores configuration for UDW which content types are allowed to be chosen from UDW, that's why we filter possible content types in search. And I think it should stay the same.
I think the problem is in the configuration/listeners themselves.